src/Test.FunctionalTests.BinSkim.Driver/BaselineTests.cs

```diff
@@ -110,8 +110,8 @@ private void RunRules(StringBuilder sb, string inputFileName)
                 SarifOutputVersion = Sarif.SarifVersion.Current,
                 TargetFileSpecifiers = new string[] { inputFileName },
                 Traces = Array.Empty<string>(),
-                Level = new List<FailureLevel> { FailureLevel.Error, FailureLevel.Warning, FailureLevel.Note, FailureLevel.None },
```

Comment:
>FailureLevel.None [](http://example.com/codeflow?start=110&length=17)

So, why did you remove none? Please explain. Either this value is useless (defined as zero) or we're missing test cases. I *think* the SARIF spec has a specific meaning for none (to help with debug messages)...

Reply:
I checked the spec, you can remove this from now. Please read the spec on this area.